docs/source/upgrade/upgrade-guide-from-monitoring-3.x-to-monitoring-4.y.rst

```diff
@@ -135,7 +134,7 @@ Rollback to version 3.x
 To rollback during the testing mode, follow `Killing the new 4.y Monitoring stack in testing mode`_ as explained previously
 and the system will continue to operate normally.
 
-To rollback to version 3.x after you completed moving to version 4.y (as shown above), run:
+To rollback to version 3.x, run the following commands after you have completed moving to version 4.y (as shown above):
```

Comment:
there's two options for rollback depends were you are at in the process, this is when you complete the upgrade,
the new version doesn't say what it should say

Reply:
@lujogre Here you need to restore the previous version because the meaning was changed (and I misunderstood the instructions after the change).

Reply:
Hello @annastuchlik , thanks for clarification.

Reply:
This is still wrong, you can roll back at any time, but the way you roll back is different if you have completed moving to 4.0 or if you are still testing.

This is the case after you completed